### DISCUSSION TITLE
Bangla OCR

### DISCUSSION THREAD

**Discussion by Automatorbd** [PaddlePaddle/PaddleOCR#14218] [Q&A]
How can i use paddle ocr to extract bangla text.

**Comment by UserWangZz** [comment]
to train a recognition model to rec the bangla text

**Reply by Automatorbd** [reply]
i have tried to train but failed is there any updated documentation

**Reply by GreatV** [reply]
please refer to https://paddlepaddle.github.io/PaddleOCR/latest/en/ppocr/model_train/training.html
